<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>609</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Nirmala UI Semilight</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MxPCR</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="ResetButton">
      <property name="font">
       <font>
        <family>Nirmala UI Semilight</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Detener</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="botonEnvioDatos">
      <property name="font">
       <font>
        <family>Nirmala UI Semilight</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Comenzar</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QComboBox" name="comboBox">
      <property name="font">
       <font>
        <family>Nirmala UI Semilight</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>PCR</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Heat Block</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mantener a 4°C</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_6">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QCustomPlot" name="customPlot" native="true">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelImage">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">image: url(:/MyFiles/flowRoot8858.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelHeatBlock">
         <property name="styleSheet">
          <string notr="true">image: url(:/MyFiles/path14626.png);</string>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelEnfriar">
         <property name="styleSheet">
          <string notr="true">image: url(:/MyFiles/flowRoot18047.png);</string>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="5">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>Nirmala UI Semilight</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Número de ciclos</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Nirmala UI Semilight</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="text">
          <string>Desnaturalización (°C)</string>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="QLineEdit" name="InputTempPrimer">
         <property name="font">
          <font>
           <family>Nirmala UI Semilight</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Nirmala UI Semilight</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Alineación  (°C)</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLineEdit" name="InputTempDenat">
         <property name="font">
          <font>
           <family>Nirmala UI Semilight</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="0" column="8">
        <widget class="QLineEdit" name="InputTempAnneal">
         <property name="font">
          <font>
           <family>Nirmala UI Semilight</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="3" column="6">
        <widget class="QLineEdit" name="InputNumCiclos">
         <property name="font">
          <font>
           <family>Nirmala UI Semilight</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="1" column="10">
        <widget class="QLineEdit" name="FinalExtensionTime">
         <property name="font">
          <font>
           <family>Nirmala UI Semilight</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="0" column="7">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Nirmala UI Semilight</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Extensión  (°C)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="9">
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <family>Nirmala UI Semilight</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Tiempo Extensión Final</string>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QLabel" name="PrimerTimeLabel">
         <property name="font">
          <font>
           <family>Nirmala UI Semilight</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Tiempo Alineación</string>
         </property>
        </widget>
       </item>
       <item row="1" column="7">
        <widget class="QLabel" name="AnnealTimeLabel">
         <property name="font">
          <font>
           <family>Nirmala UI Semilight</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Tiempo Extensión</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="DenatTimeLabel">
         <property name="font">
          <font>
           <family>Nirmala UI Semilight</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Tiempo Desnaturalización</string>
         </property>
        </widget>
       </item>
       <item row="1" column="8">
        <widget class="QLineEdit" name="AnnealTime">
         <property name="font">
          <font>
           <family>Nirmala UI Semilight</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QLineEdit" name="DenatTime">
         <property name="font">
          <font>
           <family>Nirmala UI Semilight</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="1" column="6">
        <widget class="QLineEdit" name="PrimerTime">
         <property name="font">
          <font>
           <family>Nirmala UI Semilight</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="InitDenatTime">
         <property name="font">
          <font>
           <family>Nirmala UI Semilight</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="InitDenatTimeLabel">
         <property name="font">
          <font>
           <family>Nirmala UI Semilight</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Tiempo inicial desnaturalización (seg)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QGroupBox" name="HeatBlockBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="3">
        <widget class="QLineEdit" name="InputTiempoHeatBlock">
         <property name="font">
          <font>
           <family>Nirmala UI Semilight</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="InputTempHeatBlock">
         <property name="font">
          <font>
           <family>Nirmala UI Semilight</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <family>Nirmala UI Semilight</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Tiempo</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <family>Nirmala UI Semilight</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Temperatura Heat Block</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QGroupBox" name="LowTempBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Para iniciar, da click en Comenzar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>897</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+K</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
